src/providers/item-property-data.ts

```diff
@@ -11,23 +11,47 @@ export class ItemPropertyData {
     return this.api.get(Links.brand);
   }
 
-  createBrand(body: Object) {
+  createBrand(body: any) {
```

Comment:
Would it be better practice to create an interface to use instead of `any` here and below?

Reply:
Technically yes, but I have felt like creating interfaces for every types of entities is not worth the time. We could use the ones declared in `src/models` used with ngrx, but they represent entities received from the api, which are often different than the ones sent to the api (i.e. responses often have an `organizationID` while requests don't).

I am not sure if it would still be considered a good practice to use those, but creating new ones would make it confusing and time consuming to maintain.

Reply:
I see, that makes sense!